notebooks/Planning_engine_demo.ipynb

```diff
@@ -1,320 +1 @@
n","move(l1, l2)\n"]},{"ename":"","evalue":"","output_type":"error","traceback":["\u001b[1;31mFailed to start the Kernel. \n","Traceback (most recent call last):\n","  File \"/home/luca/.vscode/extensions/ms-toolsai.jupyter-2022.4.1021342353/pythonFiles/vscode_datascience_helpers/jupyter_daemon.py\", line 158, in _start_notebook\n","    from notebook import notebookapp as app\n","  File \"/home/luca/.local/lib/python3.8/site-packages/notebook/notebookapp.py\", line 79, in <module>\n","    from .services.contents.manager import ContentsManager\n","  File \"/home/luca/.local/lib/python3.8/site-packages/notebook/services/contents/manager.py\", line 17, in <module>\n","    from nbformat import sign, validate as validate_nb, ValidationError\n","  File \"/home/luca/.local/lib/python3.8/site-packages/nbformat/sign.py\", line 26, in <module>\n","    from traitlets import (\n","ImportError: cannot import name 'Callable' from 'traitlets' (/usr/lib/python3/dist-packages/traitlets/__init__.py)\n","\n","During handling of the above exception, another exception occurred:\n","\n","Traceback (most recent call last):\n","  File \"/home/luca/.vscode/extensions/ms-toolsai.jupyter-2022.4.1021342353/pythonFiles/vscode_datascience_helpers/daemon/daemon_python.py\", line 54, in _decorator\n","    return func(self, *args, **kwargs)\n","  File \"/home/luca/.vscode/extensions/ms-toolsai.jupyter-2022.4.1021342353/pythonFiles/vscode_datascience_helpers/jupyter_daemon.py\", line 111, in m_exec_module_observable\n","    self._start_notebook(args, cwd, env)\n","  File \"/home/luca/.vscode/extensions/ms-toolsai.jupyter-2022.4.1021342353/pythonFiles/vscode_datascience_helpers/jupyter_daemon.py\", line 160, in _start_notebook\n","    from notebook import app as app\n","ImportError: cannot import name 'app' from 'notebook' (/home/luca/.local/lib/python3.8/site-packages/notebook/__init__.py)\n","\n","Failed to run jupyter as observable with args notebook --no-browser --notebook-dir=\"/home/luca/aiplan4eu/upf/notebooks\" --config=/tmp/67f562a5-f711-4f6c-a815-4318cd138d63/jupyter_notebook_config.py --NotebookApp.iopub_data_rate_limit=10000000000.0. \n","View Jupyter <a href='command:jupyter.viewOutput'>log</a> for further details."]}],"source":["with env.factory.OneshotPlanner(name='yoloplanner') as p:\n","    result = p.solve(problem)\n","    if result.status == PlanGenerationResultStatus.SOLVED_SATISFICING:\n","        print(f'{p.name} found a valid plan!')\n","        print(f'The plan is: {result.plan}')\n","        print('\\n'.join(str(x) for x in result.plan.actions))\n","    else:\n","        print('No plan found!')"]}],"metadata":{"colab":{"collapsed_sections":[],"name":"Planning Engine Demo.ipynb","provenance":[]},"interpreter":{"hash":"31f2aee4e71d21fbe5cf8b01ff0e069b9275f58929596ceb00d14d90e3e16cd6"},"kernelspec":{"display_name":"Python 3.8.10 64-bit","language":"python","name":"python3"},"language_info":{"name":"python","version":"3.8.10"}},"nbformat":4,"nbformat_minor":0}
```

Comment:
Try to keep the same formatting, in order to understand the real changes.

Execution output should not be part of the committed notebook.